java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -3739,12 +3739,16 @@ void testChunkedPackBasic() {
       }
     }
   }
-/*
+
   @Test
   void testChunkedPackTwoPasses() {
     // this test packes ~2MB worth of long into a 1MB bounce buffer
     // this is 3 iterations because of the validity buffer
     Long[] longs = new Long[256*1024];
+    for (int i = 0; i < longs.length; i++) {
+      // fill the column and set every other value to null
+      longs[i] = i % 2 == 0 ? null : (long)i;
+    }
```

Comment:
Since null is the default value for object references we could simply skip even positions 

```suggestion
    // Initialize elements at odd-numbered indices
    for (int i = 1; i < longs.length; i += 2) {
      longs[i] = (long)i;
    }
```

Reply:
Will the untouched values of `longs` be uninitialized (containing garbage value)?

Reply:
https://docs.oracle.com/javase/specs/jls/se21/html/jls-4.html#jls-4.12.5
...
Each class variable, instance variable, or array component is initialized with a default value when it is created

for all reference types (§4.3), the default value is null.
...